zephyr/esp32s3/CMakeLists.txt

```diff
@@ -5,6 +5,8 @@ if(CONFIG_SOC_SERIES_ESP32S3)
   zephyr_compile_options(-fstrict-volatile-bitfields)
   zephyr_compile_definitions_ifndef(CONFIG_MCUBOOT CONFIG_APP_BUILD_USE_FLASH_SECTIONS)
   zephyr_compile_definitions_ifdef(CONFIG_MCUBOOT BOOTLOADER_BUILD)
+  zephyr_compile_definitions_ifdef(CONFIG_ESP_SIMPLE_BOOT AND CONFIG_ESPTOOLPY_FLASHMODE_QIO BOOTLOADER_BUILD)
```

Comment:
I this case why not just leaving BOOTLOADER_BUILD enabled?
`zephyr_compile_definitions_ifdef(CONFIG_ESP_SIMPLE_BOOT BOOTLOADER_BUILD)`